libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/FormatUtils.java

```diff
@@ -55,8 +55,25 @@ public static final String formatFileSize(long size, final String[] unitStrings)
      * returns the passed double percentage (0 to 1) formatted as an human readable percentage. Ex: 0.25 returns 25%
      */
     public static final String formatPercentage(double value) {
+        return formatPercentageLimit100(value, false);
+    }
+
+    /*
+     * returns the passed double percentage (0 to 1) formatted as an human readable percentage. Ex: 0.251 returns 25.1%
+     * if limit100 is true, it limits the percentage to 100%
+     */
+    public static final String formatPercentageLimit100(double value, boolean limit100) {
+        double limit = 1.0001;
```

Comment:
I'm wondering why the limit isn't just "1" :-)

Reply:
I think I was having in mind that since we are setting fraction digits to be 1 the decimal number that matters is not the 3rd but the 4th, so well...maybe no strong reasons actually but hope should not hurt 🙃

Reply:
I think using "1" would be better but this also works 👍